java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/accessor/impl/numeric/ArrowFlightJdbcBitVectorAccessorTest.java

```diff
@@ -64,7 +62,8 @@ public void tearDown() {
     this.vectorWithNull.close();
   }
 
-  private <T> void iterate(final Function<ArrowFlightJdbcBitVectorAccessor, T> function,
+  private <T> void iterate(final AccessorTestUtils.AccessorIterator
+      .CheckedFunction<ArrowFlightJdbcBitVectorAccessor, T> function,
```

Comment:
Remove the prefixed words from CheckedFunction. You need to make changes on the import to do this